src/layer.yaml

```diff
@@ -1,24 +1,12 @@
-"includes":
-- "layer:options"
-- "layer:basic"
-- "interface:tls-certificates"
```

Comment:
don't we need this interface anymore for example? Are we getting it indirectly through one of the layers you kept?

Reply:
I'm basing this file off the cinder-purestorage charm, which I think is the one that resembles this one the most.

Reply:
but maybe we don't need TLS for purestorage but we do for netapp? I don't know the answer, just checking we don't remove things by accident.

Reply:
I believe that's unlikely, as this charm is 'configuration-only' - i.e: It only provides an interface for `cinder_configuration` and does nothing beyond that.